pkg/security/adproto/v1/activity_dump.proto

```diff
@@ -39,6 +39,7 @@ message ProcessActivityNode {
     repeated FileActivityNode files = 4;
     repeated DNSNode dns_names = 5;
     repeated SocketNode sockets = 6;
+    repeated uint32 syscalls = 7;
```

Comment:
How do we differentiate an `x64` syscall, with a `x86` syscall for example ?

Reply:
More than that, maybe we should add the arch to the metadata

Reply:
We don't and you're right. 😭

Reply:
done ✅